### ISSUE TITLE
Lots of log with Demo Picker : Called getWidth() on a recycle()'d bitmap! This is undefined behavior!

### ISSUE THREAD

**Issue by jscti** [skydoves/colorpicker-compose#55]
Hi there

**Please complete the following information:**
- Library Version : 1.0.7
- Affected Device(s) : Pixel 6 @ Android14
 
With a  simple HSV Picker without any bitmap :  

```
val controller = rememberColorPickerController()
HsvColorPicker(
                initialColor= value,
                modifier = Modifier
                    .fillMaxWidth()
                    .height(450.dp)
                    .padding(10.dp),
                controller = controller,
                onColorChanged = { colorEnvelope: ColorEnvelope ->
                   
                }
            )
```

I get a ton of logs for each touch/drag in the picker zone :  
```
Called getWidth() on a recycle()'d bitmap! This is undefined behavior!
Called getHeight() on a recycle()'d bitmap! This is undefined behavior!
Called getWidth() on a recycle()'d bitmap! This is undefined behavior!
Called getHeight() on a recycle()'d bitmap! This is undefined behavior!
Called getWidth() on a recycle()'d bitmap! This is undefined behavior!
Called getHeight() on a recycle()'d bitmap! This is undefined behavior!
Called getWidth() on a recycle()'d bitmap! This is undefined behavior!
Called getHeight() on a recycle()'d bitmap! This is undefined behavior!
Called getWidth() on a recycle()'d bitmap! This is undefined behavior!
Called getHeight() on a recycle()'d bitmap! This is undefined behavior!
```

Except this logcat pollution, everything works like a charm :)

**Comment by sderr** [comment]
Hello, I had the same issue and I think I fixed the issue in this pull request: 

https://github.com/skydoves/colorpicker-compose/pull/56

**Comment by skydoves** [comment]
Hi everyone, a new version, 1.0.8, has been released, and this issue has been fixed on #56.
